orttraining/orttraining/python/training/_utils.py

```diff
@@ -203,3 +203,21 @@ def state_dict_full_precision_key():
     """Returns the full precision key name in the state dictionary"""
 
     return 'fp32'
```

Comment:
looking at https://github.com/microsoft/onnxruntime/pull/5803, looks like there is a mismatch: 'fp32' v/s 'full_precision' returned by backend

Reply:
Thanks for pointing this out. I was under the impression that it was going to be ```fp32```. Which is what I highlighted in the checkpoint document as well (and was also highlighted as such in the backend document). Let's sync up to see what we should call it.

Reply:
I didn't get that impression from the pull request. Unless I misread the PR. It would appear that the backend document also still references the key to be ```fp32```. I am keeping this as is since this is unclear to me. We can sync up on this one.

Reply:
I updated to use ```full_precision```.